include/flatbuffers/util.h

```diff
@@ -60,6 +60,21 @@ template<> inline std::string NumToString<signed char>(signed char t) {
 template<> inline std::string NumToString<unsigned char>(unsigned char t) {
   return NumToString(static_cast<int>(t));
 }
+#if defined(FLATBUFFERS_CPP98_STL)
+template <>
+inline std::string NumToString<long long>(long long t) {
+  char buf[21]; // (log((1 << 63) - 1) / log(10)) + 2
```

Comment:
How would we do that?  log AFAIK is not constexpr.  Even if it had constexpr support in some STLs what about those that don't?